twentytwentyone-blocks/functions.php

```diff
@@ -0,0 +1,268 @@
+				),
+				array(
+					'name'  => esc_html__( 'Green', 'twentytwentyone-blocks' ),
+					'slug'  => 'green',
+					'color' => $green,
+				),
+				array(
+					'name'  => esc_html__( 'Blue', 'twentytwentyone-blocks' ),
+					'slug'  => 'blue',
+					'color' => $blue,
+				),
+				array(
+					'name'  => esc_html__( 'Purple', 'twentytwentyone-blocks' ),
+					'slug'  => 'purple',
+					'color' => $purple,
+				),
+				array(
+					'name'  => esc_html__( 'Red', 'twentytwentyone-blocks' ),
+					'slug'  => 'red',
+					'color' => $red,
+				),
+				array(
+					'name'  => esc_html__( 'Orange', 'twentytwentyone-blocks' ),
+					'slug'  => 'orange',
+					'color' => $orange,
+				),
+				array(
+					'name'  => esc_html__( 'Yellow', 'twentytwentyone-blocks' ),
+					'slug'  => 'yellow',
+					'color' => $yellow,
+				),
+				array(
+					'name'  => esc_html__( 'White', 'twentytwentyone-blocks' ),
+					'slug'  => 'white',
+					'color' => $white,
+				),
+			)
+		);
+
+		add_theme_support(
+			'editor-gradient-presets',
+			array(
+				array(
+					'name'     => esc_html__( 'Purple to Yellow', 'twentytwentyone-blocks' ),
+					'gradient' => 'linear-gradient(160deg, ' . $purple . ', ' . $yellow . ')',
+					'slug'     => 'purple-to-yellow',
+				),
+				array(
+					'name'     => esc_html__( 'Yellow to Purple', 'twentytwentyone-blocks' ),
+					'gradient' => 'linear-gradient(160deg, ' . $yellow . ', ' . $purple . ')',
+					'slug'     => 'yellow-to-purple',
+				),
+				array(
+					'name'     => esc_html__( 'Green to Yellow', 'twentytwentyone-blocks' ),
+					'gradient' => 'linear-gradient(160deg, ' . $green . ', ' . $yellow . ')',
+					'slug'     => 'green-to-yellow',
+				),
+				array(
+					'name'     => esc_html__( 'Yellow to Green', 'twentytwentyone-blocks' ),
+					'gradient' => 'linear-gradient(160deg, ' . $yellow . ', ' . $green . ')',
+					'slug'     => 'yellow-to-green',
+				),
+				array(
+					'name'     => esc_html__( 'Red to Yellow', 'twentytwentyone-blocks' ),
+					'gradient' => 'linear-gradient(160deg, ' . $red . ', ' . $yellow . ')',
+					'slug'     => 'red-to-yellow',
+				),
+				array(
+					'name'     => esc_html__( 'Yellow to Red', 'twentytwentyone-blocks' ),
+					'gradient' => 'linear-gradient(160deg, ' . $yellow . ', ' . $red . ')',
+					'slug'     => 'yellow-to-red',
+				),
+				array(
+					'name'     => esc_html__( 'Purple to Red', 'twentytwentyone-blocks' ),
+					'gradient' => 'linear-gradient(160deg, ' . $purple . ', ' . $red . ')',
+					'slug'     => 'purple-to-red',
+				),
+				array(
+					'name'     => esc_html__( 'Red to Purple', 'twentytwentyone_blocks_block_editor_script' ),
+					'gradient' => 'linear-gradient(160deg, ' . $red . ', ' . $purple . ')',
+					'slug'     => 'red-to-purple',
+				),
+			)
+		);
+
+		// Add support for responsive embedded content.
+		add_theme_support( 'responsive-embeds' );
+
+		// Add support for custom line height controls.
+		add_theme_support( 'custom-line-height' );
```

Comment:
I guess this supports could now be active with theme.json? Would you be able to confirm @nosolosw ?